### ISSUE TITLE
Support customization of the `[Unit]` section 

### ISSUE THREAD

**Issue by brouberol** [mhutter/ansible-docker-systemd-service#46]
I'm using this role to have systemd start docker containers, and some of these containers need a Samba share to be mounted locally. These mounts are described in `/etc/fstab`, causing systemd to automatically create associated `.mount` unit configurations. I'd like systemd to only start the container _after_ the mount unit.

To do this, we'd need to be able to support an additional `after` parameter, empty by default, that would allow us to supercharge the unit `after` clause.

WDYT? I'm ready to provide the patch if you think this is a good idea. Thanks!

**Comment by brouberol** [comment]
(Although, reading https://serverfault.com/a/812589, I think we'd rather be using `Requires` instead of `After`)

**Comment by mhutter** [comment]
Or mabye create something akin to the existing `service_systemd_options` but for the `[Unit]` section?

```j2
[Unit]
{% for item in unit_systemd_options %}
{{ item['key'] }}={{ item['value'] }}
{% endfor %}
```

**Comment by brouberol** [comment]
Indeed that'd be even better. We'd need to have the current block as default values, to avoid breaking existing installations. Your suggestion feels more general-purpose than what I initially had in mind, which is a plus.

**Comment by brouberol** [comment]
Hi! I see that the `2.8.0` tag was pushed to github ~a month ago, but the associated version isn't reflected on Ansible Galaxy. Would it be possible to make sure the new version is published there as well? Thank you very much!

**Comment by mhutter** [comment]
Thanks for the heads-up, it seems Ansible Galaxy is not handling the branch rename (master -> main) well. I fixed it and v2.8.0 is now on Galaxy!

**Comment by brouberol** [comment]
Thank you!
